pkg/http/api.go

```diff
@@ -350,6 +367,44 @@ func (a *api) onDeleteState(c *routing.Context) error {
 	return nil
 }
 
+func (a *api) onGetSecret(c *routing.Context) error {
+	if a.secretStores == nil || len(a.secretStores) == 0 {
```

Comment:
would you add unit-test for this ? please check api_test.go

Reply:
@youngbupark, I added some tests in 23ff198